src/test/java/org/folio/rest/api/RequestExpirationApiTest.java

```diff
@@ -191,14 +189,14 @@ public void canExpireAnFirstOpenUnfilledRequest()
       .create(),
       requestStorageUrl());
 
-    /* Status "Open - Awaiting pickup" and request expiration date in the past - should NOT be expired */
+    /* Status "Open - In transit" and request expiration date in the past - should NOT be expired */
     createEntity(
       new RequestRequestBuilder()
       .hold()
       .withId(id1_2)
       .withItemId(itemId1)
       .withPosition(2)
-      .withStatus(OPEN_NOT_YET_FILLED)
+      .withStatus(OPEN_IN_TRANSIT)
```

Comment:
Just curious, does changing to this status demonstrate the problem with the old code and now works with the new code?

Reply:
No, unfortunately i wasn't able to reproduce scenario described in your comment
Add "In transit" requests to ensure it works as expected.